packages/cactus-plugin-ledger-connector-besu/src/main/typescript/web-services/invoke-contract-endpoint.ts

```diff
@@ -87,7 +87,9 @@ export class InvokeContractEndpoint implements IWebServiceEndpoint {
     this.log.debug(reqTag);
     const reqBody: InvokeContractV1Request = req.body;
     try {
-      const resBody = await this.options.connector.invokeContract(reqBody);
+      const resBody = await this.options.connector.getContractInfoKeychain(
```

Comment:
@TonyRowntree Based on the method name getContractInfoKeychain I would conclude that the invoke contract endpoint does not do any contract invocation. Does it actually do contract invocation? If yes it definitely needs a name that implies that.